jiant/preprocess.py

```diff
@@ -235,18 +236,20 @@ def _build_vocab(args, tasks, vocab_path: str):
 
 def build_indexers(args):
     indexers = {}
-    if not args.input_module.startswith("bert") and args.input_module not in ["elmo", "gpt"]:
+    if args.input_module in ["scratch", "glove", "fastText"]:
         indexers["words"] = SingleIdTokenIndexer()
-    if args.input_module == "elmo":
+    elif args.input_module == "elmo" or args.input_module == "elmo-chars-only":
```

Comment:
`args.input_module.startswith('elmo')` ?

Reply:
Simplifying a bit, but not inclined toward startswith—we're treating BERT/XLNet models as an open class (as soon as something is added to pytorch_transformers, we get it free), but we aren't doing anything comparable for ELMo.